pkg/config/orderer.go

```diff
@@ -187,13 +219,8 @@ func kafkaBrokersValue(brokers []string) *standardConfigValue {
 	}
 }
 
-// marshalEtcdRaftMetadata serializes etcd RAFT metadata
+// marshalEtcdRaftMetadata serializes etcd RAFT metadata.
 func marshalEtcdRaftMetadata(md *eb.ConfigMetadata) ([]byte, error) {
-	var (
-		data []byte
-		err  error
-	)
-
 	copyMd := proto.Clone(md).(*eb.ConfigMetadata)
 	for _, c := range copyMd.Consenters {
```

Comment:
I'm going to assume that the values are actually pointer references and that tests cover this. (True here and other paths in the code.)